README.md

```diff
@@ -336,12 +336,15 @@ If you are using the `sync` function, and doing your update check on app start,
 #### codePush.restartApp		
 		
 ```javascript		
-codePush.restartApp(): void;		
+codePush.restartApp(onlyIfUpdateIsPending: Boolean = false): void;		
 ```		
 		
-Immediately restarts the app. If there is an update pending, it will be presented to the end user and the rollback timer (if specified when installing the update) will begin. Otherwise, calling this method simply has the same behavior as the end user killing and restarting the process. This method is for advanced scenarios, and is primarily useful when the following conditions are true:		
+Immediately restarts the app. If there is an update pending, it will be presented to the end user and the rollback timer (if specified when installing the update) will begin. Otherwise, calling this method simply has the same behavior as the end user killing and restarting the process. If a truthy value is passed to the `onlyIfUpdateIsPending` parameter, then the app will only be restarted if there is actually a pending update waiting to be applied. Otherwise, this method call will no-op.
```

Comment:
nit: Think the expression could be improved slightly, the way it is written now sounds like it will no-op if a falsey value was passed to the `onlyIfUpdateIsPending` parameter.

Maybe something like "If a truthy value is passed to the `onlyIfUpdateIsPending` parameter, then the method call will restart the app if there is actually a pending update waiting to be applied, or no-op otherwise."